lib/cache/collections.go

```diff
@@ -844,6 +844,18 @@ func (c *certAuthority) fetch(ctx context.Context) (apply func(ctx context.Conte
 		return nil, trace.Wrap(err)
 	}
 
+	// DELETE IN 13.0.
```

Comment:
Can we create a github issue and reference it here so we have a way to track that this needs to be done?